### DISCUSSION TITLE
Redash does not connect to Microsoft SQL Azure

### DISCUSSION THREAD

**Discussion by doprado** [getredash/redash#6798] [Q&A]
I'm having trouble connecting to my SQL Server Azure database. I already updated to v10.1 and still had no success.

When I try to connect via ODBC, Nginx crashes.

My database is: Microsoft SQL Azure (RTM) - 12.0.2000.8 Oct 10 2023 09:37:59

I've tried everything, before I ran it without problems on AWS RDS. After migrating to Azure I can no longer do so.

All my other apps work, only Redash doesn't.

**Comment by Thalys02** [comment]
I also the same problem,  I can't connecting my SQL server Azure database. somebody help us ?

**Comment by SauloAlmeida** [comment]
I've the same problem! Can you help us?
